system/assets/css/admin/specific/auditTrail/index/auditTrail.less

```diff
@@ -0,0 +1,126 @@
+.message-item {
```

Comment:
One little thing to consider is to use more nesting in LESS files. Really helps organise the styling. See http://lesscss.org/